libraries/chain/controller.cpp

```diff
@@ -2191,6 +2195,7 @@ struct controller_impl {
                section.read_row(rows_for_this_tid, db);
                read_row_count.fetch_add(2u, std::memory_order_relaxed);
 
+               // utils_t::preallocate(db, rows_for_this_tid.value);
```

Comment:
I thought this would help, but upon testing it didn't seem to make a difference, so I commented it out. I thought that maybe we'd want to experiment further in the future which is why I left it commented out. In any case I probably should add a comment that it doesn't appear to make a difference.